custom/enikshay/tasks.py

```diff
@@ -86,10 +88,12 @@ def run(self):
                 adherence_update = EpisodeAdherenceUpdate(episode, self)
                 voucher_update = EpisodeVoucherUpdate(self.domain, episode)
                 test_update = EpisodeTestUpdate(self.domain, episode)
+                episode_facility_id_migration = EpisodeFacilityIDMigration(self.domain, episode)
```

Comment:
because users won't all migrate to the new app version at the same time, so cases without these case properties will still get created.